src/mode.cpp

```diff
@@ -2469,9 +2484,10 @@ void ZepMode::AddNavigationKeyMaps(bool allowInVisualMode)
     AddKeyMapWithCountRegisters(navigationMaps, { "G" }, id_MotionGotoLine);
 
     // Line Motions
-    AddKeyMapWithCountRegisters(navigationMaps, { "$" }, id_MotionLineEnd);
+    AddKeyMapWithCountRegisters(navigationMaps, { "$", "<End>" }, id_MotionLineEnd);
     AddKeyMapWithCountRegisters(navigationMaps, { "^" }, id_MotionLineFirstChar);
     keymap_add(navigationMaps, { "0" }, id_MotionLineBegin);
+    keymap_add(navigationMaps, { "<Home>" }, id_MotionLineHomeToggle);
 
```

Comment:
Isn't HOME the same as 0? i.e. MotionLineBegin.
On neovim, they do the same thing for me.

Reply:
Looks to me like the behaviour you describe is specific to VS (and notepad++; though I don't have that).  I would say that for the Vim mode, Home should be id_MotionLineBegin, but for standard mode the new one you have implemented?

Reply:
Okay, makes sense, I've made that change in a new commit.